lightly/active_learning/scorers/classification.py

```diff
@@ -29,6 +29,17 @@ def _entropy(probs: np.ndarray, axis: int = 1) -> np.ndarray:
 class ScorerClassification(Scorer):
     """Class to compute active learning scores from the model_output of a classification task.
 
+    Currently supports the following scorers:
```

Comment:
Again the scorer names do not really fit the commonly used ones. Perhaps we should create a new issue for adapting this and implementing all 4 uncertainty scorers from https://towardsdatascience.com/uncertainty-sampling-cheatsheet-ec57bc067c0b